js/apps/system/_admin/aardvark/APP/react/src/views/g6graphs/AddEdgeModal.js

```diff
@@ -0,0 +1,140 @@
+/* global arangoHelper, $ */
```

Comment:
`g6graphs` are back. Is this intended? If not, this PR should be rebased on the branch for FE-135 (or devel if that has been merged).

Reply:
Resolved within https://github.com/arangodb/arangodb/pull/18199/commits/67287f90bc471d07111ce37d4b3702ea4f3c88f0 and https://github.com/arangodb/arangodb/pull/18199/commits/92b7f5e504f26e6cc619358431d1ccbc3e75cc80